clg/clg.go

```diff
@@ -7,3 +7,5 @@
 //go:generate clggen generate --clg-dir=. --template-dir=../template
 //
 package clg
+
+// TODO create CLGs based on evaluation notes in black book
```

Comment:
We stop adding even more CLGs for this PR, because there are already lots of changes. Further work can be done in new PRs. This TODO comment should stay for a while, until the notes of my black book have been realized.